### PR TITLE
Trushna's bowling challenge

### PR DESCRIPTION
**Pull request by trushnarabadia**

### CODE REVIEW

File: bowling.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+  get score() {
+    let score = 0;
+    let rollIndex = 0;
+
+    for (let frameIndex = 0; frameIndex < 10; frameIndex++) {
+      if (this.isStrike(rollIndex)) {
+        score += this.strikeBonus(rollIndex);
+        rollIndex++;
+        continue;
```
Comment (CTJMID) [inline bowling.js:18]: good use of continue key word

File: bowling.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+class Bowling {
+  constructor() {
+    this.rolls = [];
+  };
+
+  roll(pins) {
+    this.rolls.push(pins);
+  };
+
+  get score() {
```
Comment (CTJMID) [inline bowling.js:10]: good use of 'get' key word

**Review by CTJMID — COMMENTED** [review]
Nice clean code that. does exactly what you want it. to do. I would recommend adding a few tests to prove that the code catches edge cases, and doesn't allow the game to go on (or. something like that).
Very DRY and readable code! Nice use of some new ruby concepts.
